modules/apps/data-engine/data-engine-js-components-web/src/main/resources/META-INF/resources/js/custom/form/FormView.es.js

```diff
@@ -69,14 +69,28 @@ const useFormSubmit = ({apiRef, containerRef}) => {
 							return;
 						}
 
-						Liferay.Util.submitForm(event.target);
+						const avoidSubmit =
+							event.target?.dataset.ddmFormAvoidSubmit;
```

Comment:
Where is this first set?

Reply:
In the wrapper form https://github.com/liferay-data-engine/liferay-portal/pull/416/files#diff-8c8cdc08aac294bbb2bfbb8201d5c2568e4a25ec8aad85abe3460e870f1f5d8cR99 via data attribute.

Reply:
Thanks for providing that, @boton.

Although, I believe it would be preferable to pass that flag via a new tag attribute. I think it would be less of a "backdoor" to disable the form's submission and more of a custom behavior easily found by the customer. You can call it `submittable` and have `true` as its default value in the `data-layout-renderer` tag.

Could you please make these adjustments?
Thanks in advance.

Reply:
It's a backdoor, `submittable` sounds good. I will try to add this tag attribute but maybe I will need help because the data flow is complex in DE to me 😄

Reply:
No worries, @boton. Let us know! :)